objects/image.py

```diff
@@ -4,130 +4,129 @@
 import urllib
 import re
 import yaml
-from nested_lookup import nested_lookup
 
+from nested_lookup import nested_lookup
 from utils.crawler import find_image
 from utils.crawler import get_repo_pages
 
+
 class App:
-	"""This is the App Name but easier to use mainimage, subimage for documentation
-	These attributes will be from values.yaml (from a tgz)which is derived from
-	the big index.yaml file. Keywords are taken from Chart.yaml which is not saved anywhere.
+	"""This is the App Name but easier to use mainimage, subimage for 
+	   documentation. These attributes will be from values.yaml 
+	   (from a tgz) which is derived from the big index.yaml file. 
```

Comment:
```suggestion
	   (from github) but name is derived from the big index.yaml file. 
```
Updated comments, we no longer use a tarball